### ISSUE TITLE
咨询一个代码中的问题

### ISSUE THREAD

**Issue by hanquansanren** [xunull/read-DAB-DETR#1]
在DAB-DETR中transformer.py中看到一行代码，比较疑惑，希望获得您的解答
https://github.com/xunull/read-DAB-DETR/blob/96f56b9b0c543941964cfcb52b0a223bf40a20e5/models/DAB_DETR/transformer.py#L64
这里的`pos = torch.cat((pos_y, pos_x, pos_w, pos_h), dim=2)`为什么是先写pos_y，再写pos_x呢，按照COCO数据集的定义，bbox的顺序应该是x,y,w,h。因此我认为应该写作：`(pos_x,pos_y, pos_w, pos_h)`，不知道您有没有注意到这个问题呢
